chemprop/v2/models/loss.py

```diff
@@ -257,16 +257,16 @@ def forward(self, preds: Tensor, targets: Tensor, *args) -> Tensor:
         n_tasks = targets.shape[1]
         preds = preds.reshape(len(preds), n_tasks, N_CLASSES)
         y_one_hot = torch.eye(N_CLASSES, device=preds.device)[targets.long()]
-
-        return super().forward(preds, y_one_hot)
+        
+        return DirichletMixin.forward(self, preds, y_one_hot, *args)
```

Comment:
```suggestion
        return super().forward(preds, y_one_hot, *args)
```
If you're using the `super`'s method, then it's standard to call it using the `super()` object

Reply:
When I try this, the forward method from `LossFunction` gets called first. This means that line 52 returns an error as the Loss is `NoneType` because it has not gone through DirichletMixin yet. What's the correct way to use `super()` here?